sixaxis.cpp

```diff
@@ -549,15 +537,13 @@ int set_sixaxis_led(int csk, struct dev_led led, int rumble)
             recv(csk, buf, sizeof(buf), 0);
         }
     }
-#endif
 
     /* set LEDs (final) */
     setleds[11] = ledpattern[led_n];
     if (rumble) setleds[3] = setleds[4] = setleds[5] = setleds[6] = 0x00;
     send(csk, setleds, sizeof(setleds), 0);
-#ifndef GASIA_GAMEPAD_HACKS
-    recv(csk, buf, sizeof(buf), 0);
-#endif
+    if (!gasia)
+        recv(csk, buf, sizeof(buf), 0);
```

Comment:
Same as previous comment (may break shanwan & may also not be needed for official controller).